HaulageApplication/Haulage/AdminPage.xaml.cs

```diff
@@ -9,12 +11,19 @@ public AdminPage()
 	}
     private void ManageEmployees_Clicked(object sender, EventArgs e)
     {
+        Button ManageEmployees = FindByName("ManageEmployees") as Button;
+        ManageEmployees.Clicked += async (sender, args) => { await Navigation.PushAsync(new ManageEmployees()); };
```

Comment:
The ManageEmployees_Clicked method is triggered when the ManageEmployees button is clicked (this is set up in Admin.xaml)

why does this method then add another anonymous function as a handler to the ManageEmployees click event which then issues the await Navigation.PushAsync(new ManageEmployees()) call?

Why not just call await Navigation.PushAsync(new ManageEmployees()) directly in the ManageEmployees_Clicked method?

Reply:
think this was part of aidans code to manage employees as the new button i added is on line 22-27
This still problematic code though and will fix it. Think aidan did it this way because the approach previously used didn't work as there was a name clash between the page and the button.